evm/p2p/auth.py

```diff
@@ -160,9 +161,9 @@ def decode_auth_ack_message(self, ciphertext: bytes) -> Tuple[datatypes.PublicKe
         if len(ciphertext) < ENCRYPTED_AUTH_ACK_LEN:
             raise ValueError("Auth ack msg too short: {}".format(len(ciphertext)))
         elif len(ciphertext) == ENCRYPTED_AUTH_ACK_LEN:
-            eph_pubkey, nonce, version = decode_ack_plain(ciphertext, self.privkey)
+            eph_pubkey, nonce, __ = decode_ack_plain(ciphertext, self.privkey)
```

Comment:
Any reason for using a double underscore over a single underscore?  We have an existing convention of using single underscore as the *not-used* variable.

Reply:
I found https://stackoverflow.com/questions/5893163/what-is-the-purpose-of-the-single-underscore-variable-in-python/5893946#5893946

But I'll switch to single underscore if that's the convention in this project.

Reply:
Thnx @h00701350103 , I'd like to stick with single underscore for now.